iphone/Maps/Bookmarks/BookmarksList/BookmarksListInfoViewController.swift

```diff
@@ -26,17 +26,23 @@ final class BookmarksListInfoViewController: UIViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
-    descriptionButton.setTitle(L("placepage_place_description").uppercased(), for: .normal)
     separatorsConstraints.forEach { $0.constant = 1 / UIScreen.main.scale }
 
     guard let info = info else { return }
     updateInfo(info)
   }
 
-  private func updateInfo(_ info: IBookmakrsListInfoViewModel) {
+  private func updateInfo(_ info: IBookmarksListInfoViewModel) {
     titleLabel.text = info.title
     descriptionButton.isHidden = !info.hasDescription
-
+    descriptionButton.isEnabled = info.isHtmlDescription
```

Comment:
What if a non-html description is big enough? Will it expand in the disabled state?

Reply:
It won't. I think that we should distinguish 2 use cases here:

1. A user specified an html string - in this case we should certainly allow to open a new WebView to display rich formatting, pictures, etc.
2. A user set a plain string, which I assume to be really short in majority of cases. Thus, it'd be excessive to open a WebView for this

As an option, we can 1) enable the button only if the string size will be more than it's possible to fit the string using [NSString.size(withAttributes:)](https://developer.apple.com/documentation/foundation/nsstring/1531844-size), or
2) shrink the label with [adjustsFontSizeToFitWidth](https://developer.apple.com/documentation/uikit/uilabel/1620546-adjustsfontsizetofitwidth), or
3) use ExpandableLabel only for plain strings
<img width="292" alt="image" src="https://github.com/organicmaps/organicmaps/assets/135858315/ec6d3385-bed5-4294-a9a6-39b77130ec61">

Reply:
@biodranik I reconsidered :) Now I show WKWebView for both cases

Reply:
Can you please set a small margin and 1.25 or 1.5 line interval for plain text?

Reply:
Does it look better now?

Reply:
I think it looks better
![Simulator Screenshot](https://github.com/organicmaps/organicmaps/assets/135858315/3ac93388-e9e0-4251-8c09-cc6d99439347)

Reply:
@biodranik